apps/examples/testcase/Kconfig

```diff
@@ -20,6 +20,7 @@ source "$APPSDIR/examples/testcase/le_tc/kernel/Kconfig"
 source "$APPSDIR/examples/testcase/le_tc/network/Kconfig"
 source "$APPSDIR/examples/testcase/le_tc/ttrace/Kconfig"
 source "$APPSDIR/examples/testcase/le_tc/tcp_tls/Kconfig"
+source "$APPSDIR/examples/testcase/le_tc/compression/Kconfig"
```

Comment:
Please add it by Alphabet order in le_tc tab

Reply:
Done